lib/bundler/gem_helper.rb

```diff
@@ -181,6 +182,10 @@ def sh_with_code(cmd, &block)
       end
     end
 
+    def gem_key
+      Bundler.settings['gem.push_key'].to_s.downcase if Bundler.settings['gem.push_key']
```

Comment:
Double quotes, and please add documentation for this setting in `man/bundle-config.ronn`?